tests/createProjectDriver.py

```diff
@@ -0,0 +1,136 @@
+
+from loginDriver import logIn
+from selenium.webdriver import Chrome
+import time
+
+class createProject:
+
+
+    def __init__(self):
+        self.driver = Chrome()
+    
+    def Close(self):
+        self.driver.quit()
+    
+    def setProjectName(self, projectname, projectpassword):
+        #This is to set up the projectName + description
+        self.driver.find_element_by_id("project_name").send_keys(projectname)
+        self.driver.find_element_by_id("project_description").send_keys(projectpassword)
+    
+    def download(self):
+        self.driver.find_element_by_link_text("(Download a sample roster files)").click()
+        self.driver.implicitly_wait(10)
+    
+    def setRoster(self, studentFile):
+
+        self.driver.find_element_by_id("student_file").send_keys(studentFile)
+    
+    def setRubrics(self, jsonFile):
+        self.driver.find_element_by_id("json_file").send_keys(jsonFile)
+    
+    def driver_createProject(self, username, password, projectname, projectpassword, 
```

Comment:
why are some methods starting with "driver_"?

Reply:
No methods are named this way in the current state of the test suite.